kazoo/handlers/utils.py

```diff
@@ -241,7 +241,8 @@ def create_tcp_connection(
             context.load_default_certs(ssl.Purpose.SERVER_AUTH)
             if check_hostname and not verify_certs:
                 raise ValueError(
-                    "verify_certs must be False when check_hostname is specified"
+                    "verify_certs must be False when"
```

Comment:
From the check on line 242, think the intent is to make sure that the hostname validation is *in addition* to the cert validation, correct?
If so, I think the error msg is misleading.
How about something like "verify_certs must also be True when check_hostname is True" ?

Reply:
Agreed. I have tweaked the wording a bit now.